google-cloud-storage/src/main/java/com/google/cloud/storage/Storage.java

```diff
@@ -1809,6 +1811,53 @@ Blob create(
   @Deprecated
   Blob create(BlobInfo blobInfo, InputStream content, BlobWriteOption... options);
 
+  /**
+   * Uploads the given {@code path} to the blob using {@link #writer}. By default any md5 and crc32c
+   * values in the given {@code blobInfo} are ignored unless requested via the {@code
+   * BlobWriteOption.md5Match} and {@code BlobWriteOption.crc32cMatch} options. Folder upload is not
```

Comment:
I think this should be 
`via the {@code BlobWriteOption.md5Match} and {@code BlobWriteOption.crc32cMatch}` => `via the {@link BlobWriteOption.md5Match} and {@link BlobWriteOption.crc32cMatch}`.